managers/signals.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Any
+from typing import Set
+
+from django.db.models.signals import m2m_changed
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from employees.models import TaskActivityType
+from managers.models import Project
+from users.models import CustomUser
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(m2m_changed, sender=Project.managers.through)
+def update_user_type(sender: Project, action: str, pk_set: Set, **kwargs: Any) -> None:
+    assert sender == Project.managers.through
+    project = kwargs["instance"]
+    logger.debug(f"Updates on project with id: {sender.pk} for users with id: {pk_set}")
+    if action in ["pre_remove", "post_remove"]:
+        change_user_type_to_employee(pk_set)
+    elif action in ["pre_add", "post_add"]:
+        change_user_type_to_manager(project)
+    else:
```

Comment:
this `else` seems unnecessary

Reply:
I created it because once @kbeker  suggest it to me:
https://github.com/Code-Poets/sheetstorm/pull/98#discussion_r270030292

Reply:
I would insist on removing it - it's redundant. @kbeker what would be the advantages of having it here?

Reply:
@dybi I don't remember exactly when but we discussed during Concent project how to handle cases where if-elif statemets are and we decided that we should always end statements like this with `else-return` statement. But now when I'm staring and this piece of code I think that this might be deleted I think that I made mistake about case where we should use `else-return` statement and this is sure not good place for it. So @Szymiks as @dybi suggested - please delete it.